Changelog entry for remote instance local login (#3080)
FlowFuse/website: src/changelog/2025/03/device-local-login.md

---
title: Local Login for Remote Instances
description: "Local Login option for Remote Instances, powered by Device Agent"
date: 2025-03-27 12:00:00.0  
authors: ["stephen-mclaughlin"]
tags:
  - changelog
---

> We have introduced a new Local Login option for Remote Instances.

Reviewer: ```suggestion
With the new 3.2.0 release of the FlowFuse Device Agent, we have introduced a new Local Login option for Remote Instances.
```

Highlight this requires the new device agent to be used.

> This feature allows you to enable or disable local login for you Remote Instance, which is particularly useful in air-gapped environments or times when the FlowFuse platform is unreachable.

Reviewer: ```suggestion
This feature allows you to enable a local login for your Remote Instance, which is particularly useful in air-gapped environments or times when the instance cannot reach the FlowFuse platform.
```

Slight rewording to take emphasis off the platform being unavailable - rather the device's connection being unavailable.

> When local login is enabled, users can log in to Node-RED locally using the specified credentials.
> 
> Please note that this feature is not the recommended method for accessing your Remote Instance. However, we acknowledge that some users may require this functionality for specific use cases.

Reviewer: ```suggestion
When local login is enabled, users can log in to Node-RED directly using the specified credentials.

For day-to-day activity, we still recommending accessing the editor through the FlowFuse Platform for the best user experience. However, for those times when that isn't possible, this option provides a fall-back to allow continued access to the device.
```
